### ISSUE TITLE
how to run a single test file?

### ISSUE THREAD

**Issue by nikitpatel24** [meanjs/mean#1640]
Is there any way where i could run a only a single test file for TDD development.
I can only run here `npm tun test:client` or `npm run test:server` ,These runs all the test cases. Is there a similar way to run a single test file.?

**Comment by simison** [comment]
Nope, but you can first run all server tests and then leave it watching for changes — and when you edit a test file, it'll run only modified tests. If you then edit a regular app file, it'll run all the tests again.

```
npm run test:server:watch
```

If you need the same for client side tests, you can copy the task from our project: https://github.com/Trustroots/trustroots/blob/master/gulpfile.js#L393-L400

**Comment by hyperreality** [comment]
Actually, there is. Go to `config/assets/test.js` and edit the object value so it contains only the test file you want to run.

**Comment by shanavas786** [comment]
You can use `describe.only` or `it.only` in test files to run a single/set of tests.
